tests/alpine-build/run-tests.sh

```diff
@@ -5,5 +5,5 @@ cd $1
 cd /crun
 git clean -fdx
 ./autogen.sh
-./configure CFLAGS='-Wall -Wextra -Werror' && make -j $(nproc)
+./configure CFLAGS='-Wall -Wextra -Werror' --disable-systemd && make -j $(nproc)
```

Comment:
Does alpine not have systemd? Or is the goal to just test withou systemd support?

Reply:
Alpine is an OpenRC based distro. The packages are not provided.